packages/editor/src/components/post-featured-image/index.js

```diff
@@ -150,6 +155,25 @@ function PostFeaturedImage( {
 		);
 	}
 
+	const { isRequestingFeaturedImageMedia } = useSelect(
+		( select ) => {
+			const _isRequestingFeaturedImageMedia =
+				!! featuredImageId &&
+				! select( coreStore ).hasFinishedResolution( 'getMedia', [
+					featuredImageId,
+					{ context: 'view' },
+				] );
+
+			return {
+				isRequestingFeaturedImageMedia: _isRequestingFeaturedImageMedia,
+			};
+		},
+		[ featuredImageId ]
+	);
```

Comment:
Let's select the resolution status in `withSelect` below. We're already selecting media there, and grouping these makes sense. Plus, we'll use existing store subscriptions.